qa/tasks/cephfs/mount.py

```diff
@@ -166,7 +165,8 @@ def assert_and_log_minimum_mount_details(self):
                               sudo=True).decode())
 
     def is_mounted(self):
-        return self.mounted
+        return self.client_remote.run(args=f'findmnt {self.hostfs_mntpt}',
```

Comment:
What will happen if `mntpt` is mounted by different filesystems or filesystem other than ceph?
Do we need to check only ceph-fuse here?

Reply:
It can't be so because `hostfs_mntpt` is a variable from qa.tasks.cephfs.mount.CephFSMount. Thefore for it can't be any other FS.